docs/product-business-resource-planner-guide/src/main/asciidoc/optimizer-maven-configuration-proc.adoc

```diff
@@ -0,0 +1,40 @@
+[id='optimizer-maven-configuration-proc']
+= Setting up {PLANNER} with Maven, Gradle, Ivy, or Buildr
```

Comment:
Sounds rather like a sub-chapter of some "Getting Started" or "Embedding Business Optimizer into your application" chapter.

Reply:
Ok I've renamed this chapter and it probably needs to be checked again.

Reply:
@rsynek I have another issue open specifically for engine installation, which relies on this information.

I will merge this change and assign review to you for the other issue so you can review this there instead.